composer.json

```diff
@@ -21,7 +21,8 @@
     },
     "minimum-stability" : "dev",
     "require": {
-        "oat-sa/oatbox-extension-installer": "~1.1||dev-master"
+        "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
+        "ramsey/uuid": "3.8"
```

Comment:
see https://github.com/oat-sa/extension-tao-revision/pull/85/files?w=1#r372372844

Reply:
I agree, it will be difficult to manage versioning if it is explicit in each extension that requires this lib

Reply:
for the moment , for being able to merge into develop,  we need to keep it this way,
the trait need to be at the Generis extension level and  we will use it when building the package
but not inside this PR 
(the provided link is inside the spanner branch that is a long way from being merged )

Reply:
So please put this spanner PR has required, but we are not doing temp solution because correct one is one merged